src/components/NativeBalance/NativeBalance.tsx

```diff
@@ -2,16 +2,33 @@ import React, { useEffect, useState } from 'react';
 import { useMoralis } from 'react-moralis';
 import NativeBalanceStyles from './NativeBalance.styles';
 import { NativeBalanceProps } from './types';
-
 const { BalanceStyled } = NativeBalanceStyles;
 
 const NativeBalance: React.FC<NativeBalanceProps> = ({ style }) => {
     const { account, chainId, web3, Moralis } = useMoralis();
     const [balance, setBalance] = useState<{
         formatted?: string;
         balance?: unknown;
+        ticker?: string;
     }>({});
 
+    function ticker(chain: string) {
```

Comment:
we have `getNativeByChain` in `src/web3uitils/networks.ts`

https://github.com/web3ui/web3uikit/blob/master/src/web3utils/networks.ts#L85

Reply:
ok, did not see this, will edit